### ISSUE TITLE
[Web] [Accessibility] [11yMAS][Vision Impairment][MAS4.3.1] [Focus indicator is not visible on “Filter by product” combo box in High contrast modes]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#691]
**User Experience:**
User with vision impairment will not be able to know where the focus is present if focus indicator is not visible on “Filter by product” control in High contrast modes.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools Used: High contrast White/Black modes
**Attachments:**
[MAS4.3.1_Focus not visible.zip](https://github.com/OfficeDev/script-lab/files/1445424/MAS4.3.1_Focus.not.visible.zip)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Turn on High contrast settings and Navigate to “Filter by product” combo box  .
3. Verify whether focus indicator is visible on “Filter by product” combo box in both High contrast white/Black modes or not. 
 
**Current Behavior:** 
Focus indicator is not visible on “Filter by product” combo box in High contrast modes.

**Expected Behavior:**
Focus indicator should be visible on “Filter by product” combo box in High contrast modes, so that user will be able to see where focus is available.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.3.1
